app/models/rbac.rb

```diff
@@ -484,7 +484,8 @@ def self.search(options = {})
   end
 
   def self.method_with_scope(ar_scope, options)
-    if ar_scope.respond_to?(:instances_are_derived?) ? ar_scope.instances_are_derived? : (ar_scope < ActsAsArModel)
+    # in all reality, these extend ActsAsArModel
+    if ar_scope.respond_to?(:instances_are_derived?) && ar_scope.instances_are_derived?
```

Comment:
could this be simplified to something like `if ar_scope.try(:instances_are_derived?)`